jbpm-runtime-manager/src/test/java/org/jbpm/runtime/manager/impl/PerProcessInstanceRuntimeManagerTest.java

```diff
@@ -1568,7 +1568,7 @@ public void testExceptionWhenCompleteTaskAfterEngineDisposal() throws Exception
         emf.close();
     }
 
-    @Test(timeout=15000)
+    @Test //(timeout=15000)
```

Comment:
```suggestion
    @Test
```

Reply:
I have created a test case here:
https://github.com/kiegroup/jbpm/commit/837f19b7b0ead384c1c982f1e7e733eb4e3bac39

For the use case of a signal being sent without a processInstanceId, the additional SessionInfo entry is created as a consequence of this code:
https://github.com/kiegroup/jbpm/blob/main/jbpm-runtime-manager/src/main/java/org/jbpm/runtime/manager/impl/PerProcessInstanceRuntimeManager.java#L164-L172

@elguardian the code change from this PR does not seem to address this.